thecodingmachine/graphqlite-bundle/3.0/config/packages/graphqlite.yaml

```diff
@@ -0,0 +1,16 @@
+# Read more about Graphqlite available options at: https://graphqlite.thecodingmachine.io/docs/symfony-bundle
+graphqlite:
+    namespace:
+        controllers: App\Controller\
+        types:
+            - App\
+    debug:
+        # Include exception messages in output when an error arises
+        INCLUDE_DEBUG_MESSAGE: false
```

Comment:
You should only provide non default values here.

The comments should be in your bundles Configuration via “->info()”-node if not already done.

Off topic and not relevant for the recipe:
Config keys should be lowercase, otherwise one could think it’s an env var. Or is there any good reason behind it which I doesn’t know?

Reply:
Hey @OskarStark ,

Thanks a lot for the feedback.
I added the `->info()` and will remove the non-default values.

The keys are in upper-case because they relate to config constants in a bit-mask.

I did not find another way to do this, but maybe there is one?

For instance, if you put

```yaml
debug:
    INCLUDE_DEBUG_MESSAGE: true
    INCLUDE_TRACE: true
```

the bundle will make it similar to writing this (in Webonix\GraphQL):

```php
use GraphQL\Error\Debug;
$debug = Debug::INCLUDE_DEBUG_MESSAGE | Debug::INCLUDE_TRACE;
```

Ideally, I would like to be able to write something like:

```yaml
debug: INCLUDE_DEBUG_MESSAGE | INCLUDE_TRACE
```

but I'm not sure how to do that in Symfony.

Reply:
> You should only provide non default values here.

I would like you to address Oskars comment. We dont need these default values here.

Reply:
@Nyholm @OskarStark : sorry for the delay. It's fixed.